graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/FilesResource.java

```diff
@@ -38,9 +70,73 @@ public Response getCqlDirectives() {
         .build();
   }
 
+  @GET
+  @Path("/namespace/{namespaceName}.graphql")
+  @Produces(MediaType.TEXT_PLAIN)
+  public Response getSchema(
+      @HeaderParam("X-Cassandra-Token") String token,
+      @PathParam("namespaceName") String namespace,
+      @QueryParam("version") String version,
```

Comment:
Are versions user defined or an incremental number? Just wondering if this should be an int or if there should be some sort of other validation.

Reply:
Oh, looks like a UUID? In that case should probably validate it and return an appropriate error message rather than bubbling up the 500.

Reply:
Versions are timeuuids.

Reply:
> In that case should probably validate it and return an appropriate error message rather than bubbling up the 500

Indeed. I returned a 404 as well because versions are opaque strings as far as users are concerned.